great_expectations/render/renderer/content_block/content_block.py

```diff
@@ -31,13 +31,17 @@ class ContentBlockRenderer(Renderer):
     _default_element_styling = {}
 
     @classmethod
-    def validate_input(cls, render_object):
+    def validate_input(cls, render_object: Any) -> None:
         pass
 
     @classmethod
-    def render(cls, render_object, **kwargs):
+    def render(
+        cls, render_object: Any, **kwargs
+    ) -> Union[_rendered_component_type, Any, None]:
         cls.validate_input(render_object)
-        exception_list_content_block = kwargs.get("exception_list_content_block")
+        exception_list_content_block: Optional[bool] = kwargs.get(
```

Comment:
This Optional[bool] plagues all of our downstream internal methods. If we don't get the value here, can we set it to False and get rid of the Optional bit?

Reply:
sounds good! don't love the Optional pattern we sometimes have but I'm wary of changing things in case we rely on the nullability of the value.

As long as tests pass, I think keeping this as a strict bool is good!